snowflake_udf_demo/skyflow_udf_demo.sql

```diff
@@ -0,0 +1,46 @@
+-- Step 0: Set default database to use. Please ensure Skyflow Setup worksheet has been completed.
+USE DATABASE SKYFLOW_DEMO;
+
+-- Step 1: Check table to see initial plain text PII
+SELECT * FROM DOCTORS_DEMO;
+
+-- Step 2: Tokenize the table, passing in table name as a parameter, and query the table again.
+CALL SKYFLOW_TOKENIZE_TABLE('DOCTORS_DEMO');
+
+-- Step 3a: Run a sample query without detokenizing, for example the doctors who started after year 2010. PII will be tokenized.
+SELECT  DOCTOR_ID,
+        NAME,
+        EMAIL,
+        SPECIALTY,
+        EMPLOYMENT_START_DATE
+FROM DOCTORS_DEMO
+WHERE EMPLOYMENT_START_DATE > '2010';
+
+-- Step 3b: Now run the query using Skyflow detokenize to obtain PII data.
+SELECT  DOCTOR_ID,
+        SKYFLOW_DETOKENIZE(NAME),
+        SKYFLOW_DETOKENIZE(EMAIL),
+        SPECIALTY,
+        EMPLOYMENT_START_DATE
+FROM DOCTORS_DEMO
+WHERE EMPLOYMENT_START_DATE > '2010';
+
+-- Step 4: Make some updates to the data
+DELETE FROM DOCTORS_DEMO WHERE DOCTOR_ID = 1 OR DOCTOR_ID = 2; -- Delete a few records
+UPDATE DOCTORS_DEMO SET NAME = 'New Name', PHONE = '212-555-5555' WHERE DOCTOR_ID = 3; -- Update a record
+INSERT INTO DOCTORS_DEMO VALUES -- Insert a few new records
+    (101, 'John Smith', 'john@example.com', '718-444-5555', '123 Fake Street NY NY 10019', 'Cardiology', '2020-01-01'),
```

Comment:
@samsternberg this is a bit of a nitpick, but would you mind changing these example numbers to all use a 555? Just want to avoid any potential issues around people thinking these are real phone numbers (which technically they might be?).

Reply:
Great use of `example.com` and 555 elsewhere though!

Reply:
@jstjoe Done!